Libraries/Picker/PickerIOS.ios.js

```diff
@@ -44,21 +46,49 @@ var PickerIOS = React.createClass({
   _stateFromProps: function(props) {
     var selectedIndex = 0;
     var items = [];
+    var styleFromProps = this._getStyleFromProps(props);
+
+    var itemsStyle = {
+      fontSize: styleFromProps.fontSize || 20,
+      color: styleFromProps.color || '#000000',
+      textAlign: styleFromProps.textAlign || 'center'
+    };
+
     ReactChildren.forEach(props.children, function (child, index) {
       if (child.props.value === props.selectedValue) {
         selectedIndex = index;
       }
-      items.push({value: child.props.value, label: child.props.label});
+      items.push({
+        value: child.props.value, 
+        label: child.props.label, 
+        style: itemsStyle
+      });
     });
     return {selectedIndex, items};
   },
 
+  _getStyleFromProps: function(props){
+    var styleFromProps;
+
+    if(typeof props.style === 'number') {
```

Comment:
I'm pretty sure it's not necessary to do all this. You can combine pre-baked stylesheets with prop styles by using an array, like this:

     style={[styles.pickerIOS, this.props.style]}